### PR TITLE
{lang}[foss/2020a] rstudio v1.3.1093

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3102.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/de943dab9917219ecf9b4009942de371 for a full test report.

**Review by Micket — COMMENTED** [review]
This is probably a bit of a stretch, but, maybe rstudio could be made independent of `R`, moved up to GCCcore level, and just let leave it up to the user to load whichever version of R they want.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node352.hydra.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/983328b6685f518ab6f83dd575cd3a39 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'rstudio'
+version = '1.3.1093'
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+
+homepage = 'https://www.rstudio.com/'
+description = """This RStudio Server version.
```
Comment (ocaisa) [inline easybuild/easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb:8]: Me Tarzan, you Jane

File: easybuild/easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'rstudio'
```
Comment (ocaisa) [inline easybuild/easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb:3]: I know history has brought us to this point, but Shouldn't this be called `RStudio-Server`?

Comment (boegel) [reply easybuild/easyconfigs/r/rstudio/rstudio-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb:3]: Yep, and you fixed that in #11764 , thanks!

**Pull request merged**